app/modules/teams/resources.py

```diff
@@ -14,13 +14,14 @@
 from app.extensions.api.parameters import PaginationParameters
 from app.modules.users import permissions
 from app.modules.users.models import User
+from flask.ext.restplus_patched._http import HTTPStatus
```

Comment:
This is a deprecated way of importing flask extentions. Please, just use `from flask_restplus_patched...` and also put it in the previous group of imports.